base/codeinsights-db/codeinsights-db.Deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    description: Code Insights TimescaleDB instance.
+  labels:
+    deploy: sourcegraph
+    sourcegraph-resource-requires: no-cluster-admin
```

Comment:
Please add `app.kubernetes.io/component` labels to all the manifests here. See https://github.com/sourcegraph/deploy-sourcegraph/blob/0eaab6588cd270cc536beda0101a7ad681d167e6/base/frontend/sourcegraph-frontend.Ingress.yaml#L15

Reply:
Should I add this to `base/codeintel-db/codeintel-db.Deployment.yaml`? It's not in there either

Reply:
That would be great if you could, thanks. It's not just just the deployment manifests though, it's all files related to the component. ￼